src/bundle/Resources/views/themes/admin/ui/component/dropdown/dropdown.html.twig

```diff
@@ -0,0 +1,117 @@
+        {{ custom_init|default(false) ? "ibexa-dropdown--custom-init" }}
+        {{ no_items ? "ibexa-dropdown--no-items" }}
+        {{ class|default('') }}
+    "
+>
+    <div class="ibexa-dropdown__source">
+        {{ source|default(null)|raw }}
+    </div>
+    <div class="ibexa-dropdown__wrapper">
+        <ul
+            class="ibexa-dropdown__selection-info"
+            data-template="{{ include('@ibexadesign/ui/component/dropdown/dropdown_selected_item.html.twig', {
+                value: '{{ value }}',
+                label: '{{ label }}',
+            })|e('html_attr') }}"
+
+        >
+            {% if no_items %}
+                <li class="ibexa-dropdown__selected-item ibexa-dropdown__selected-item--predefined ibexa-dropdown__selected-placeholder">
+                    {{ 'dropdown.placeholder.empty'|trans|desc("No options available") }}
+                </li>
+            {% else %}
+                {% for choice in choices_flat %}
+                    {% if custom_form ? choice.value == value : choice is selectedchoice(value) %}
+                        {% include '@ibexadesign/ui/component/dropdown/dropdown_selected_item.html.twig' with {
+                            value: choice.value,
+                            label: _self.get_translated_label(choice.label, translation_domain),
+                        } %}
+                    {% endif %}
+                {% endfor %}
+
+                {% if value is empty %}
+                    {% if multiple %}
+                        <li class="ibexa-dropdown__selected-item ibexa-dropdown__selected-item--predefined ibexa-dropdown__selected-placeholder">
+                            {{ _self.get_translated_placeholder(placeholder, translation_domain) }}
+                        </li>
+                    {% else %}
+                        {% if placeholder is not none %}
+                            {% include '@ibexadesign/ui/component/dropdown/dropdown_selected_item.html.twig' with {
+                                value: '',
+                                label: _self.get_translated_label(placeholder, translation_domain),
+                            } %}
+                        {% else %}
+                            {% set first_choice = choices_flat|first %}
+
+                            {% include '@ibexadesign/ui/component/dropdown/dropdown_selected_item.html.twig' with {
+                                value: first_choice.value,
+                                label: _self.get_translated_label(first_choice.label),
+                            } %}
+                        {% endif %}
+                    {% endif %}
+                {% endif %}
+            {% endif %}
+
+            <li class="ibexa-dropdown__selected-item ibexa-dropdown__selected-item--predefined ibexa-dropdown__selected-overflow-number" hidden></li>
+        </ul>
+        <div class="ibexa-dropdown__items">
+            {% if choices_flat|length >= min_search_items %}
```

Comment:
I wonder, wouldn't it be better to hide search with CSS class? Then you can always change decision in JS.